recipes/zeromq/all/conanfile.py

```diff
@@ -22,7 +22,7 @@ class ZeroMQConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
-        "encryption": [None, "libsodium", "tweetnacl"],
+        "encryption": [None, "none", "libsodium", "tweetnacl"], # None for conan v1 compat, use "none" in conan v2 for no encryption
```

Comment:
See https://github.com/conan-io/conan-center-index/issues/17289#issuecomment-1526500000 for my rational :)

```suggestion
        "encryption": [False, "libsodium", "tweetnacl"],
```

Reply:
This would break v1 I guess, why not `"encryption": [None, False, "libsodium", "tweetnacl"],` ?

Reply:
Break how? Boolean of None is False. Please, let's keep it simpler, only False is enough for this case.